### ISSUE TITLE
Error inside the powershell window

### ISSUE THREAD

**Issue by KrazyKleb** [Zgoly/bloxstrap-multi-instance-integration#1]
```
New-Object : Exception calling ".ctor" with "2" argument(s): "A WaitHandle with system-wide name
'ROBLOX_singletonEvent' cannot be created. A WaitHandle of a different type might have the same name."
At C:\Users\User\AppData\Local\Bloxstrap\Integrations\Multi-instance\Multi-instance.ps1:153 char:18
+         $mutex = New-Object System.Threading.Mutex($true, $mutexName)
+                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [New-Object], MethodInvocationException
    + FullyQualifiedErrorId : ConstructorInvokedThrowException,Microsoft.PowerShell.Commands.NewObjectCommand
```
Also I can't use multiple instances

**Comment by Zgoly** [comment]
Interesting. I'll look later to see what the problem might be

**Comment by Zgoly** [comment]
@KrazyKleb @LaffX Thanks for your reports. Most likely, the reason was the recent Bloxtrap update, after which Roblox began to launch faster due to code optimization.

Most likely because [this line of code](https://github.com/bloxstraplabs/bloxstrap/blob/v2.7.0/Bloxstrap/Bootstrapper.cs#L404) is missing in the new version of Bloxtrap.

I'm already looking for a solution.

**Comment by mosy-png** [comment]
I also seem to get the same error now. Also seems like i cant downgrade bloxtrap anymore after the new update of it

**Comment by KrazyKleb** [comment]
The roblox account manager Multi Roblox setting also stopped working

**Comment by Zgoly** [comment]
@mrrewilh I'm working on a replacement right now.

Unfortunately, I will have to abandon the idea of using a program like Integration (of course for the time when Bloxstrap will function in this way). I went as far as changing the registry and running Bloxstrap through my script, but this solution turned out to be very unstable. 

But this is not a reason to be upset, because the new solution will be more convenient, advanced and prettier than the current one.

I'm planning to release it the next day.

**Comment by mrrewilh** [comment]
@Zgoly Thank you for explaining the situation instead of being rude even though I commented with rude language. I know we need to talk about codes and such on a platform like this. But I still wanted to thank you.

**Comment by xyzopen1** [comment]
How do i change the path file it keep looking bloxstrap on drive C while it on Drive D

**Comment by Axel-ctirl** [comment]
hey can you give me the link to it when your done with the new one

**Comment by Zgoly** [comment]
> How do i change the path file it keep looking bloxstrap on drive C while it on Drive D

If you mean the current script, it expects Blockstrap to be located exclusively in the appdata folder (for some reason I didn't think of this). I think I can fix this by getting the path from the registry.

**Comment by Zgoly** [comment]
> hey can you give me the link to it when your done with the new one

Of course! I will ping everyone with a link who was interested in the update

**Comment by Zgoly** [comment]
@KrazyKleb @LaffX @mosy-png @Axel-ctirl @mrrewilh @xyzopen1

The project is complete.

Before you begin, please remove the current integration using [Uninstall.bat](https://github.com/Zgoly/bloxstrap-multi-instance-integration/blob/main/Uninstall.bat) to avoid conflicts.

## [You can find all the information and download the project here](https://github.com/Zgoly/MultiBloxy).

I would be VERY grateful if you could give this new project a couple of stars. It will help with search engine rankings and on GitHub itself.

This issue will remain open.

**Comment by Axel-ctirl** [comment]
Hey is there a possibility that you can make a translator for Roblox(the one that Roblox provides don't work)
